src/components/elements/input/TimeDurationInput.tsx

```diff
@@ -0,0 +1,65 @@
+import { Typography } from '@mui/material';
+import { Stack } from '@mui/system';
+import { memoize } from 'lodash-es';
+import { useMemo } from 'react';
+import NumberInput from '@/components/elements/input/NumberInput';
+import { TextInputProps } from '@/components/elements/input/TextInput';
+
+type Props = { value?: number; onChange?: (val: number | null) => void } & Omit<
+  TextInputProps,
+  'value' | 'onChange'
+>;
+
+const TimeDurationInput = ({ value, onChange, label, ...props }: Props) => {
+  const [hours, minutes] = useMemo(() => {
```

Comment:
I got into a weird state where the hours were always prefixed with a "0" and I couldn't make it go away. I think it happens if you do this:
1. select minutes field, type a number
2. select hour field (which is now 0) and try to type a number

If we switch to using `0` as placeholders (See other comment), I think we should just leave hours as empty if there are no hours selected. That should get rid of the funkiness.

Reply:
Yes, this is better! I also added ` || ''` to the value prop for each of the inputs, so it should render as blank instead of 0 when there is no value for minutes or hours.